runtime/lua/vim/health.lua

```diff
@@ -1,5 +1,58 @@
 local M = {}
 
+-- Indents lines *except* line 1 of a string if it contains newlines.
+function M._indent_after_line1(s, columns)
+  local lines = vim.split(s, '\n')
+  for i, _ in ipairs(lines) do
+    if i ~= 1 then
+      local indent = ''
+      for _ = 1, columns do
+        indent = ' ' .. indent
+      end
+      lines[i] = indent .. lines[i]
+    end
+  end
+  return table.concat(lines, '\n')
+end
```

Comment:
This is a more robust approach:
 
```lua
function M._indent_after_line1(s, columns)
  local lines = vim.split(s, '\n')
  local indent = string.rep(' ', columns)
  for i = 2, #lines do
    lines[i] = lines[i]:gsub('^%s*', indent)
  end
  return table.concat(lines, '\n')
end
```

If you want to not indent blank lines, use `^%s*%S` instead of `^%s*`. If you want to not indent only empty lines, use `^%s+`.

Reply:
Hmm, the string.rep is nice. Thank you!